internal/pkg/describe/webapp.go

```diff
@@ -22,6 +22,18 @@ type WebAppURI struct {
 	Path    string // Empty if the application is served on HTTPS. Otherwise, the pattern used to match the application.
 }
 
+// WebAppECSParams contains ECS deploy parameters of a web application.
+type WebAppECSParams struct {
+	TaskSize
+	ContainerPort string
+	TaskCount     string
+}
+
+type TaskSize struct {
```

Comment:
nit: can be private

Reply:
If that one is private then cannot be used in CLI package which is required in `app_show_test.go`